src/nidigital/examples/configure_time_set_and_voltage_levels/nidigital_configure_time_set_and_voltage_levels.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
+
+import argparse
+import nidigital
+import os
+import sys
+
+
+class VoltageLevelsAndTerminationConfig():
+    def __init__(self, vil, vih, vol, voh, vterm, termination_mode, iol, ioh, vcom):
+        self.vil = vil
+        self.vih = vih
+        self.vol = vol
+        self.voh = voh
+        self.vterm = vterm
+        self.termination_mode = termination_mode
+        self.iol = iol
+        self.ioh = ioh
+        self.vcom = vcom
+
+
+class TimeSetConfig():
+    def __init__(self, time_set_name, period, drive_format, drive_on, drive_data, drive_return, drive_off, strobe_edge):
+        self.time_set_name = time_set_name
+        self.period = period
+        self.drive_format = drive_format
+        self.drive_on = drive_on
+        self.drive_data = drive_data
+        self.drive_return = drive_return
+        self.drive_off = drive_off
+        self.strobe_edge = strobe_edge
+
+
+def convert_drive_format(drive_format):
+    converter = {'NR': nidigital.DriveFormat.NR,
+                 'RL': nidigital.DriveFormat.RL,
+                 'RH': nidigital.DriveFormat.RH,
+                 'SBC': nidigital.DriveFormat.SBC}
+    return converter.get(drive_format, None)
+
+
+def example(resource_name, options,
```

Comment:
nitpick: Each param in its own line or all in one line

Reply:
Fixed.